incubator/mongodb/README.md

```diff
@@ -0,0 +1,192 @@
+# MongoDB Helm Chart
+
+## Prerequisites Details
+* Kubernetes 1.3 with alpha APIs enable
+* PV support on the underlying infrastructure
```

Comment:
Do we need PVC?  Could we run emptydir?  I can think of a couple of use cases.  I know nutty right.